cluster/operator.go

```diff
@@ -77,3 +99,26 @@ func (o Operator) HashTreeRootWith(hh *ssz.Hasher) error {
 
 	return nil
 }
+
+// randomName returns a deterministic name for an ecdsa public key. The name consists of a noun
+// and an adjective separated by a hyphen. The noun is calculated using PublicKey's X coordinate
+// while the adjective is calculated using PublicKey's Y coordinate.
+func randomName(pk ecdsa.PublicKey) string {
```

Comment:
lets move this function to its own file maybe `name.go`? and split out the lists to global variables defined one word per line.